docs/use-case-descriptions/message-validation.md

```diff
@@ -13,25 +13,73 @@ If it fails validation, a response message is sent with ‘rejected’ in the re
-| Time zone rejected               | The message specifies a time zone that has a different UTC offset than is the agreed- upon common value for the market.                                                                                               |
-| Invalid congestion point         | Unknown congestion point or the recipient is not active at this congestion point.                                                                                                                                     |
-| Unknown reference <Reference\>  | The message with the sequence where is referred to is unknown.</br>For <Reference\> the concerning reference field name can be filled in (for example FlexRequestSequence or PrognosisSequence).                     |
-| Reference Period mismatch        | The message(s) with the sequence where is referred to contains a different Period.                                                                                                                                    |
-| Reference message expired        | The message that is referred to is expired.                                                                                                                                                                           |
-| Reference message revoked        | The message that is referred to is revoked.                                                                                                                                                                           |
-| ISPs out of bounds               | One or more ISPs are outside the tolerated boundaries: ISPs do not exist.                                                                                                                                             |
-| ISP conflict                     | One or more ISPs are defined more than once, possibly because of an incorrect duration.                                                                                                                               |
-| Period out of bounds             | Period of the message is inappropriate.</br>For example: a FlexRequest with a Period in the past or a settlement item in a FlexSettlement with a Period outside the concerning settlement period.                     |
-| ExpirationDateTime out of bounds | ExpirationDateTime is in the past or exceeds the ISPs in the message.                                                                                                                                                 |
+
+| RejectionReason | Cause of rejection | Applicable to messages |
+|---|---|---|
+| Unknown SenderDomain | SenderDomain specified in the SignedMessage is unknown to the receiving implementation. | SM |
+| Invalid SenderRole | The SenderRole specified in the SignedMessage is invalid. | SM |
+| Invalid Sender combination | The combination SenderDomain and SenderRole in the SignedMessage is invalid.| SM |
```

Comment:
Would it make sense to replace `Invalid Sender combination` with something liike `Unknown Sender`

Reply:
We thought about this, it might indeed make sense. As we thoyght about it our idea was: There is a difference between Unknown Sender (a sender that can't be resolved and is really unknown) and an Invalid Sender combination; in that case you can resolve part of the information (like the sender name) but the combination is not correct (the role might be DSO while the sender is an AGR).

It might be a good thing to discuss this one in the TSC.